ga4gh/datarepo.py

```diff
@@ -385,6 +388,11 @@ def _readSystemTable(self, cursor):
         row = cursor.fetchone()
         self._repositoryVersion = config["repositoryVersion"]
```

Comment:
yes to schemaVersion

Jerome Kelleher <notifications@github.com> writes:
> In ga4gh/datarepo.py:
> 
> > @@ -385,6 +388,11 @@ def _readSystemTable(self, cursor):
> >          row = cursor.fetchone()
> >          self._repositoryVersion = config["repositoryVersion"]
> 
> I was thinking about this, perhaps it should be schemaVersion? Repository
> version seems to imply that we're somehow tracking all the data that's going in
> and out. What do you think?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub*
>